ted_sws/notice_metadata_processor/services/notice_eligibility.py

```diff
@@ -18,22 +42,29 @@ def check_package(mapping_suite: MappingSuite, notice_metadata: NormalisedMetada
     :param mapping_suite:
     :return:
     """
+
     constraints = mapping_suite.metadata_constraints.constraints
 
     eform_subtype = notice_metadata.eforms_subtype
     notice_publication_date = datetime.datetime.fromisoformat(notice_metadata.publication_date)
-    notice_xsd_version = notice_metadata.xsd_version
 
-    end_date = constraints[END_DATE_KEY][0] if constraints[END_DATE_KEY] else datetime.datetime.now().isoformat()
-    constraint_start_date = datetime.datetime.fromisoformat(constraints[START_DATE_KEY][0])
-    constraint_end_date = datetime.datetime.fromisoformat(end_date)
-    constraint_min_xsd_version = constraints[MIN_XSD_VERSION_KEY][0]
-    constraint_max_xsd_version = constraints[MAX_XSD_VERSION_KEY][0]
+    if notice_metadata.is_eform:
+        notice_xsd_version = notice_metadata.eform_sdk_version
+        eforms_sdk_version = notice_xsd_version.rsplit('-', 1)[1]
```

Comment:
this split need to be explained, maybe we need to use a template or regex selector instead of unclear split.